trustymail/domain.py

```diff
@@ -58,7 +62,7 @@ class Domain:
     def __init__(self, domain_name, timeout, smtp_timeout, smtp_localhost, smtp_ports, smtp_cache, dns_hostnames):
         self.domain_name = domain_name.lower()
 
-        self.base_domain_name = public_list.get_public_suffix(self.domain_name)
+        self.base_domain_name = get_public_suffix(self.domain_name)
```

Comment:
Something odd is happening here.  For the domain `dhs.gov`, for example, this line gives `gov` for `self.base_domain_name`.  We actually want `dhs.gov` as `self.base_domain_name` in this case.

This line also gives `gov` as the base domain for `www.dhs.gov`, which is definitely not the intent.  It also leads to incorrect results in the input, since obviously `.gov` has no DMARC record and hence `trustymail` reports that there is no DMARC record on the base domain.

If I start the python CLI, though, I get:
```python
Python 3.6.4 (default, Dec 23 2017, 19:07:07) 
[GCC 7.2.1 20171128] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import publicsuffix
>>> psl = publicsuffix.PublicSuffixList()
>>> psl.get_public_suffix('www.dhs.gov')
'dhs.gov'
>>> psl.get_public_suffix('dhs.gov')
'dhs.gov'
```
These are the expected results.  I don't know why different results are returned in the code.